docs/how-to-guides/data-basics/5-adding-a-delete-button.md

```diff
@@ -0,0 +1,454 @@
+
+Luckily, the `SnackbarList` WordPress component can help us with all of that. It presents a list of notifications in an accessible, user-friendly manner. That's precisely the component used in the Widgets editor and in the rest of WordPress!
+
+Here's how we can use it:
+
+```js
+import { SnackbarList } from '@wordpress/components';
+import { store as noticesStore } from '@wordpress/notices';
+
+function SnackbarNotices() {
+	const notices = []; // We'll come back here in a second!
+
+	return (
+		<SnackbarList
+			notices={ snackbarNotices }
+			className="components-editor-notices__snackbar"
+		/>
+	);
+}
+```
+
+That's all great, but the notices list is empty. How do we populate it? We'll lean on the same package as WordPress: [`@wordpress/notices`](https://github.com/WordPress/gutenberg/blob/895ca1f6a7d7e492974ea55f693aecbeb1d5bbe3/docs/reference-guides/data/data-core-notices.md).
+
+Here's how:
+
+```js
+import { SnackbarList } from '@wordpress/components';
+import { store as noticesStore } from '@wordpress/notices';
+
+function SnackbarNotices() {
+	const notices = useSelect(
+		( select ) => select( noticesStore ).getNotices(),
+		[]
+	);
+	const { removeNotice } = useDispatch( noticesStore );
+	const snackbarNotices = notices.filter( ({ type }) => type === 'snackbar' );
+
+	return (
+		<SnackbarList
+			notices={ snackbarNotices }
+			className="components-editor-notices__snackbar"
+			onRemove={ removeNotice }
+		/>
+	);
+}
+
+function MyFirstApp() {
+	// ...
+	return (
+		<div>
+			{/* ... */}
+			<SnackbarNotices />
+		</div>
+	);
+}
+```
+
+This tutorial is focused on managing the pages and won't discuss the above snippet in detail. If you're interested in the details of `@wordpress/notices`, the [handbook page](https://developer.wordpress.org/block-editor/reference-guides/data/data-core-notices/) is a good place to start.
+
+Now we're ready to tell the user about any errors that may have occurred.
+
+#### Dispatching notifications
+
+With the SnackbarNotices component in place, we're ready to dispatch some notifications! Here's how:
+
+```js
+import { store as noticesStore } from '@wordpress/notices';
+import { useEffect } from '@wordpress/element';
+function DeletePageButton( { pageId } ) {
+	const { createSuccessNotice, createErrorNotice } = useDispatch( noticesStore );
+	// useSelect returns a list of selectors if you pass the store handle
+	// instead of a callback:
+	const { getLastEntityDeleteError } = useSelect( coreDataStore )
+	const handleDelete = async () => {
+		const success = await deleteEntityRecord( 'postType', 'page', pageId);
+		if ( success ) {
+			// Tell the user the operation succeeded:
+			createSuccessNotice( "The page was deleted!", {
+				type: 'snackbar',
+			} );
+		} else {
+			// We use the selector directly to get the error at this point in time.
```

Comment:
This still mysterious and I think the comment makes things worse :)

Reply:
I agree, I replaced it with

```
			// We use the selector directly to get the fresh error *after* the deleteEntityRecord
			// have failed.
```